front/src/applications/editor/tools/trackSplit/components/TrackSplitLeftPanel.tsx

```diff
@@ -86,7 +86,7 @@ const TrackSplitLeftPanel: FC<unknown> = () => {
             id="offset"
             type="number"
             inputProps={{ required: true }}
-            value={state.offset / 1000}
+            value={state.offset === 0 ? '' : state.offset / 1000}
```

Comment:
If we forbid them to write 0 we need to change the tooltip as well 
![Screenshot_20240628_113436](https://github.com/OpenRailAssociation/osrd/assets/24652545/6e0c24ef-358c-4131-9d66-ab69af87bc0c)